src/inference/dev_api/openvino/runtime/icompiled_model.hpp

```diff
@@ -151,11 +149,13 @@ class OPENVINO_RUNTIME_API ICompiledModel : public std::enable_shared_from_this<
     friend ov::IExecutableNetworkWrapper;
     friend InferenceEngine::ICompiledModelWrapper;
 
+protected:
+    std::vector<ov::Output<const ov::Node>> m_inputs;
+    std::vector<ov::Output<const ov::Node>> m_outputs;
```

Comment:
Why do you need this change?
get_inputs() and get_outputs() methods are virtual, you can override them

Reply:
Not used now